### PR TITLE
Tests E2E: Update E2E test 'InstallCheByOperatorHub' according to changes on UI

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Update locators and methods to have an install application on both OCP 4.1 and OCP 4.2 and reduce an addiction on changes of UI in next releases OCP
* Do refactor and update methods in page objects
* Implement log into installed application with switched on _OpenShif oAuth_ property by using regular user of OCP
* Update and modify parameters in the _TetsConstants_

### What issues does this PR fix or reference?
#15464

### CODE REVIEW

File: tests/e2e/pageobjects/openshift/OcpLoginPage.ts [hunk 1]
```diff
@@ -42,6 +41,27 @@ export class OcpLoginPage {
         await this.driverHelper.waitAndClick(loginWithKubeAdminLocator);
     }
 
+    async clickOnLoginWitnHtpasswd() {
+        Logger.debug('OcpLoginPage.clickOnLoginWitnHtpasswd');
```
Comment (dmytro-ndp) [inline tests/e2e/pageobjects/openshift/OcpLoginPage.ts:45]: Do we really need `Logger.debug()` in final version?

**Review by Ohrimenko1988 — APPROVED** [review]

File: tests/e2e/pageobjects/openshift/OcpLoginPage.ts (hunk 1 above)
Comment (dmytro-ndp) [reply tests/e2e/pageobjects/openshift/OcpLoginPage.ts:45]: Yes, we do :-)

**Review by dmytro-ndp — COMMENTED** [review]
Which set of parameters should be applied to `docker run` command to run the test:
1) using Eclipse Che operator
2) using CRW operator

Can you, please, provide an example of command to run the test?

**Comment by artaleks9** [comment]
@dmytro-ndp 
* Used command: _npm run test-che-install_
* Parameters:
  - TS_SELENIUM_BASE_URL = url of deployed application
  - TS_SELENIUM_WEB_CONSOLE_OCP_URL = url web console OpenShift
  - TS_SELENIUM_OPERATORHUB_PAGE_URL = url OperatorHub main page = `${TS_SELENIUM_WEB_CONSOLE_OCP_URL}/operatorhub/all-namespaces`

  - TS_SELENIUM_CATALOG_TILE_OPERATOR_NAME =>
        - Eclipse Che = `eclipse-che-preview`
        - CRW = `codeready-workspaces-latest` or should replace on `codeready-workspaces-preview`?

  - TS_SELENIUM_OPERATOR_LOGO_NAME =>
        - Eclipse Che = `Eclipse Che`
        - CRW = `Red Hat CodeReady Workspaces`

  - TS_SELENIUM_INSTALL_PROJECT_NAME = target namespace to install application
  - TS_OCP_OPERATOR_UPDATE_CHANNEL =>
         - Eclipse Che = `nightly`
         - CRW = `final`
  - TS_SELENIUM_VALUE_TLS_SUPPORT = `true` or `false`
  - TS_SELENIUM_VALUE_SELF_SIGN_CERT = `true` or `false`
  - TS_SELENIUM_VALUE_OPENSHIFT_OAUTH = `true` or `false`
  - TS_SELENIUM_INSTALL_APP_PREFIX_URL (it seems this parameter is redundant in test) =>
          - Eclipse Che = `che`
          - CRW = `codeready`
  - TS_SELENIUM_USERNAME = username to login to installed application, default `admin`
  - TS_SELENIUM_PASSWORD = password to login to installed application, default `admin`
  - TS_OCP_LOGIN_PAGE_HTPASW = `false` by default; `true` if configured an HTPasswd identity provider

  - TS_SELENIUM_MULTIUSER = `false` by default; `true` if configured MultiUser mode
  - TS_SELENIUM_LOGIN_CHE_OCP_USER = `false` by default; `true` if login to installed application by using regular user OCP

  - TS_SELENIUM_OCP_TEMP_ADMIN_USERNAME = temp admin username OCP, `kubeadmin` by default
  - TS_SELENIUM_OCP_TEMP_ADMIN_PASSWORD = temp admin password
  - TS_SELENIUM_OCP_USERNAME = regular username OCP
  - TS_SELENIUM_OCP_PASSWORD = password regular user OCP

**Comment by dmytro-ndp** [comment]
Taking into account that there could be installation using DevConsole as well, I would propose to rename test to `test-operatorhub-installation`, to leave the gap for `test-devconsole-installation`.

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**